main/sdr.go

```diff
@@ -365,7 +384,9 @@ func sdrWatcher() {
 				if err != nil {
 					serial = ""
 				}
-				if strings.Compare(serial, "stratux:1090") != 0 {
+
+				r, _ := regexp.Compile("str?a?t?u?x:1090")
```

Comment:
The regexp string isn't changing so no reason to compile it with each pass, just do it once on entry in to the function prior to dropping in to the for loop. Why no error check/handling?